[TEST-FAILURE] client.map.ClientMapTest.testPutIfAbsentTtl

```
java.lang.AssertionError: expected:<value3> but was:<null>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:834)
    at org.junit.Assert.assertEquals(Assert.java:118)
    at org.junit.Assert.assertEquals(Assert.java:144)
    at com.hazelcast.client.map.ClientMapTest.testPutIfAbsentTtl(ClientMapTest.java:358)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast-client-new/718/testReport/junit/com.hazelcast.client.map/ClientMapTest/testPutIfAbsentTtl/

New failure: https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-pr-builder/17020/testReport/junit/com.hazelcast.client.map/ClientMapTest/testPutIfAbsentTtl/

**Error Message**
```
expected:<value1> but was:<null>
```
**Stacktrace**
```
java.lang.AssertionError: expected:<value1> but was:<null>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.hazelcast.client.map.ClientMapTest.testPutIfAbsentTtl(ClientMapTest.java:397)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapTest.java
@@ -373,7 +373,7 @@ public class ClientMapTest extends HazelcastTestSupport {
     @Test
     public void testPutTtl() {
         final IMap<String, String> map = createMap();
-        map.put("key1", "value1", 1, TimeUnit.SECONDS);
+        map.put("key1", "value1", 10, TimeUnit.SECONDS);
         assertNotNull(map.get("key1"));
         assertTrueEventually(new AssertTask() {
             @Override
@@ -393,7 +393,7 @@ public class ClientMapTest extends HazelcastTestSupport {
     @Test
     public void testPutIfAbsentTtl() {
         final IMap<String, String> map = createMap();
-        assertNull(map.putIfAbsent("key1", "value1", 1, TimeUnit.SECONDS));
+        assertNull(map.putIfAbsent("key1", "value1", 10, TimeUnit.SECONDS));
         assertEquals("value1", map.putIfAbsent("key1", "value3", 1, TimeUnit.SECONDS));
         assertTrueEventually(new AssertTask() {
             @Override

